### PR TITLE
Feature/userflow components

### PR DESCRIPTION
**Pull request by lukas-menzel**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ x ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Cleanup (changes in structure but not functionality)

**Test Configuration**:
* Go version:
* Kafka version:
* Kubernetes version:

# Checklist:

- [ x ] My code follows the style guidelines of this project
- [ x ] I have performed a self-review of my own code
- [ x ] I have commented my code, particularly in hard-to-understand areas
- [ x ] I have made corresponding changes to the documentation
- [ x ] My changes generate no new warnings
- [ x ] I have added tests that prove my fix is effective or that my feature works
- [ x ] New and existing unit tests pass locally with my changes
- [ x ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: frontend/src/Dashboard.css [hunk 1]
```diff
@@ -19,7 +19,7 @@
   height: 42px;
   width: 663px;
   align-self: flex-start;
-  margin: 0;
+  margin: 0px;
```
Comment (m-lukas) [inline frontend/src/Dashboard.css:22]: Actually, both do the same if it's `0`

File: frontend/src/components/ProfileSelection.js [hunk 2]
```diff
@@ -67,25 +75,56 @@ function ProfileSelection({ apiClient, faceHits, onProfileSelect }) {
...
+            >
+              <div>
+                {profile.user.userName}: <br />
+              </div>
+              <div style={{ fontSize: 14 }}>
+                {Math.round((profile.weight / weightSum) * 100)}% confidence
+              </div>
+            </div>
+
+            {/* {uniqWith(profile.facesList, (a, b) => a.postId === b.postId).map(
```
Comment (m-lukas) [inline frontend/src/components/ProfileSelection.js]: Comment can be removed

File: frontend/src/Dashboard.css (hunk 1 above)
Comment (lukas-menzel) [reply frontend/src/Dashboard.css:22]: deleted it

File: frontend/src/components/ProfileSelection.js (hunk 2 above)
Comment (lukas-menzel) [reply frontend/src/components/ProfileSelection.js]: removed

**Pull request merged**